dotCMS/src/main/java/com/dotcms/ai/AiKeys.java

```diff
@@ -0,0 +1,66 @@
+    public static final String CONTENT = "content";
+    public static final String MODEL = "model";
+    public static final String SYSTEM = "system";
+    public static final String USER = "user";
+    public static final String TEMPERATURE = "temperature";
+    public static final String TITLE = "title";
+    public static final String SIZE = "size";
+    public static final String TYPE = "type";
+    public static final String URL = "url";
+    public static final String IMAGE = "image";
+    public static final String EMBEDDING = "embedding";
+    public static final String EMBEDDINGS = "embeddings";
+    public static final String N = "N";
+    public static final String RESPONSE = "response";
+    public static final String ERROR = "error";
+    public static final String DEFAULT = "default";
+    public static final String DATA = "data";
+    public static final String INDEX = "index";
+    public static final String INDEX_NAME = "indexName";
+    public static final String INDEX_COUNT = "indexCount";
+    public static final String FIELD_VAR = "fieldVar";
+    public static final String IDENTIFIER = "identifier";
+    public static final String TIME_TO_EMBEDDINGS = "timeToEmbeddings";
+    public static final String TOTAL_TO_EMBED = "totalToEmbed";
+    public static final String EMBEDDINGS_COUNT = "embeddingsCount";
+    public static final String DELETE_QUERY = "deleteQuery";
+    public static final String DELETED = "deleted";
+    public static final String LANGUAGE = "language";
+    public static final String INODE = "inode";
+    public static final String CONTENT_TYPE = "contentType";
+    public static final String SITE = "site";
+    public static final String CREATED = "created";
+    public static final String CONFIG_HOST = "configHost";
+    public static final String MODDATE = "moddate";
+    public static final String DISTANCE = "distance";
+    public static final String EXTRACTED_TEXT = "extractedText";
+    public static final String AVAILABLE_MODELS = "availableModels";
+    public static final String TOTAL_TIME = "totalTime";
+    public static final String SEARCH = "search";
+    public static final String MAX_TOKENS = "max_tokens";
+    public static final String STREAM = "stream";
+    public static final String OPEN_AI_RESPONSE = "openAiResponse";
+    public static final String SUPPORTING_CONTENT = "supportingContent";
+    public static final String CACHE = "cache";
+    public static final String MATCHES = "matches";
+    public static final String QUERY = "query";
+    public static final String TOTAL = "total";
+    public static final String THRESHOLD = "threshold";
+    public static final String DOT_CMS_RESULTS = "dotCMSResults";
+    public static final String OPERATOR = "operator";
+    public static final String OFFSET = "offset";
+    public static final String LIMIT = "limit";
+    public static final String COUNT = "count";
+    public static final String INPUT = "input";
```

Comment:
This is all good work @victoralfaro-dotcms but let's send this type of changes in different PRs so it's clear what the actual required changes were.

Reply:
Extracted this into a commit and removed from current branch.